lib/assets/constants.dart

```diff
@@ -33,8 +33,10 @@ final TERMS_AND_CONDITIONS_URI = Uri.parse(
 );
 
 final ABOUT_US_URI = Uri.parse('https://www.now-u.com/about');
-final FEEDBACK_FORM_URI = Uri.parse('https://forms.gle/xenzFkRVrELsqnnB9');
-final JOIN_RESEARCH_FORM_URI = Uri.parse('https://forms.gle/DjSdjH1DfNtRwXiP7');
+final FEEDBACK_FORM_URI = Uri.parse('https://www.now-u.com/app/feedback');
+final JOIN_RESEARCH_FORM_URI = Uri.parse(
+  'https://www.now-u.com/app/research/signup',
```

Comment:
Maybe either we can stick to trailing comma like here, or remove it, like value above? i think I'm voting for the second one

Reply:
Yeah that was the original plan but then the linter shouted at me and tried to do the thing where it puts the assignment on the line after: https://github.com/now-u/now-u-app/commit/5ab93ffa8baed658ef906a1d85d753dc85f435cd
I was thinking of making the above match the latter but I think if we can fit on one line it probably makes sense? Maybe I should have just accepts darts insistence on having assignments on the line after